Cannot build the recent version

I use GHC 8.2 and cannot build the recent version:

```
$ cabal install distributed-process-simplelocalnet
Resolving dependencies...
In order, the following will be installed:
network-multicast-0.2.0 (via: distributed-process-simplelocalnet-0.2.4) (reinstall) (changes: network-2.6.3.2 -> 2.6.3.1)
distributed-process-simplelocalnet-0.2.4 (new version)
Warning: Note that reinstalls are always dangerous. Continuing anyway...
Configuring network-multicast-0.2.0...
Building network-multicast-0.2.0...
Installed network-multicast-0.2.0
Configuring distributed-process-simplelocalnet-0.2.4...
Building distributed-process-simplelocalnet-0.2.4...
Failed to install distributed-process-simplelocalnet-0.2.4
Build log ( /Users/david/.cabal/logs/distributed-process-simplelocalnet-0.2.4.log ):
cabal: Entering directory '/var/folders/f5/9f4k6xqn7t59k6bfgsl4dhpr0000gp/T/cabal-tmp-8255/distributed-process-simplelocalnet-0.2.4'
Configuring distributed-process-simplelocalnet-0.2.4...
Building distributed-process-simplelocalnet-0.2.4...
Preprocessing library distributed-process-simplelocalnet-0.2.4...
[1 of 2] Compiling Control.Distributed.Process.Backend.SimpleLocalnet.Internal.Multicast ( src/Control/Distributed/Process/Backend/SimpleLocalnet/Internal/Multicast.hs, dist/build/Control/Distributed/Process/Backend/SimpleLocalnet/Internal/Multicast.o )

src/Control/Distributed/Process/Backend/SimpleLocalnet/Internal/Multicast.hs:24:36: error:
    Module ‘Network.Transport.Internal’ does not export ‘decodeNum32’

src/Control/Distributed/Process/Backend/SimpleLocalnet/Internal/Multicast.hs:24:49: error:
    Module ‘Network.Transport.Internal’ does not export ‘encodeEnum32’
cabal: Leaving directory '/var/folders/f5/9f4k6xqn7t59k6bfgsl4dhpr0000gp/T/cabal-tmp-8255/distributed-process-simplelocalnet-0.2.4'
cabal: Error: some packages failed to install:
distributed-process-simplelocalnet-0.2.4 failed during the building phase. The
exception was:
ExitFailure 1
```

--- a/distributed-process-simplelocalnet.cabal
+++ b/distributed-process-simplelocalnet.cabal
@@ -36,7 +36,7 @@ Library
                      binary >= 0.6.3 && < 0.9,
                      containers >= 0.4 && < 0.6,
                      transformers >= 0.2 && < 0.6,
-                     network-transport >= 0.4.1.0 && < 0.6,
+                     network-transport >= 0.5 && < 0.6,
                      network-transport-tcp >= 0.4 && < 0.7,
                      distributed-process >= 0.5.0 && < 0.8
   Exposed-modules:   Control.Distributed.Process.Backend.SimpleLocalnet,
@@ -61,7 +61,7 @@ Executable TestSimpleLocalnet
                        binary >= 0.5 && < 0.9,
                        containers >= 0.4 && < 0.6,
                        transformers >= 0.2 && < 0.6,
-                       network-transport >= 0.4.1.0 && < 0.6,
+                       network-transport >= 0.5 && < 0.6,
                        network-transport-tcp >= 0.4 && < 0.7,
                        distributed-process >= 0.5.0 && < 0.8
   Else

